### ISSUE TITLE
Tab right click stops working after using "Change Tab to Workspace"

### ISSUE THREAD

**Issue by ShotaroKaneda** [zen-browser/desktop#3549]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Prerequisites:
* Have multiple Workspaces (at least 2)
* have some tabs open in either iof them

Steps:
* Right click a tab and select "Change Tab to Workspace" and move it to another workspace. (this works fine)

After this, right clicking on tabs stops working in all workspaces.
One can make it work again if you right click inside a webpage to have the context menu pop up.
After doing this once, right click on tabs works again

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.0 (Firefox 133.0) (64-bit)

### What platform are you seeing the problem on?

Windows

**Comment by tarn-jihas** [comment]
Can confirm, the bug is there and the temporary 2. fix of dosubot is working.

**Comment by ShotaroKaneda** [comment]
> Can confirm, the bug is there and the temporary 2. fix of dosubot is working.

yeah. either that or right clicking inside a webpage (as detailed above) works to re-enable it for tabs.

**Comment by luke-monroe** [comment]
Can confirm the bug and the temp fix #2
